_extras/guide.md

```diff
@@ -2,37 +2,45 @@
 title: "Instructor Notes"
 ---
 
-## Introduction
```

Comment:
I would much prefer to keep the headings for the individual episodes, as I think this makes it much easier for folks to understand which part of the lesson is relevant to each of these notes. Perhaps we could have the episode names as second-level headings and the note titles you have added (which are definitely also helpful!) as third-level, i.e.

```Markdown
## Introduction

### HedgeDoc/CodiMD Collaborative Notes Document

[...]

###  'Add Your Repository Details to HedgeDoc' Exercise 

[...]

## Reusing Blocks of Content

### `includes` vs `include` Confusion

[...]

## Page Layouts

### Layout Name Confusion

[...]
```

Reply:
Ah, got you @tobyhodges - totally makes sense. I was thinking of improving this PR with links to episodes in the text (I added some of them but not for all notes) but it is much better and clearer to use the episode names as section headers and have all notes for that episode included.

Reply:
Improved via f926df171568dc98f4bb507d2d6f2c9330d1c92a